paddle/framework/backward.cc

```diff
@@ -28,15 +28,15 @@ namespace paddle {
 namespace framework {
 
 static inline std::unique_ptr<OperatorBase> CreateGradOp(
-    const OperatorBase& op,
-    const std::unordered_set<std::string>& no_grad_set) {
+    const OperatorBase& op, const std::unordered_set<std::string>& no_grad_set,
+    std::unordered_map<std::string, std::string>* grad_name_map) {
```

Comment:
maybe `grad_name_map` ==> `grad_to_var`? or some better name.

Reply:
Actually, we do not need change runtime implementation. But, just leave it there is OK.